ConnectorActionableMsgs/01 Build a basic Connector/Demos/01-card-webhook/Readme.md

```diff
@@ -1,44 +1,46 @@
-# Demo - Section 1: Create a Card payload and submit it via an Incoming Webhook
+# Demo - Create a Card payload and submit it via an Incoming Webhook
+
+In this demo, you will copy an example card from the Message Card Playground website and send it to an Office 365 group in your tenant.
 
 To run this demo, perform the following steps from the lab:
 
-## Exercise 1: Create a Card payload and submit it via an Incoming Webhook
-In this exercise, you will copy an example card from the Message Card Playground web site and send it to an Office 365 Group in your tenant.
+## Create a group and configure the webhook
+
+1. Open a browser and go to [Microsoft Outlook](https://outlook.office.com). Log in with your Office 365 credentials.
+1. Create a new group, or select an existing group.
+1. At the top-left of the screen, select the gear next to the member account. Choose **Connectors**.
 
-### Create Group and configure Webhook
+    ![Screenshot of group settings menu](../../Images/Exercise1-01.png)
 
-1. Open a browser and navigate to **https://outlook.office.com**. Log in with your Office 365 credentials.
-1. Create a new Group, or navigate to an existing Group.
-1. At the top-left of the screen, click the gear next to the member count. Choose **Connectors**.
+1. In the list of connectors, scroll down and select **Incoming Webhook**. Select **Add**.
 
-    ![](../../Images/Exercise1-01.png)
+    ![Screenshot of list of Connectors with incoming webhook highlighted](../../Images/Exercise1-02.png)
 
-1. In the list of Connectors, scroll down and then select **Incoming Webhook**. Click **Add**.
+1. Enter a name for the connector and select **Create**.
+1. The page will re-display, now including a URL for the connector. Use the icon next to the URL to copy it to the clipboard.
 
-    ![](../../Images/Exercise1-02.png)
+    ![Screenshot of incoming webhook with URL](../../Images/Exercise1-03.png)
 
-1. Enter a name for the connector and click **Create**.
-1. The page will re-display, now including a URL for the connector. Click the icon next to the URL to copy it to the clipboard.
+    > Note: The URL will be used several times in this exercise, as will the clipboard. We recommend pasting the URL into Notepad or other application.
 
-    ![](../../Images/Exercise1-03.png)
+1. Select **Done**. Then close the Connector and Group settings menus.
 
-    > Note: The URL will be used several times in this Exercise, as will the clipboard. We recommend pasting the URL into Notepad or other application.
+## Explore MessageCard playground
```

Comment:
We are now calling it "Card Playground" instead of "MessageCard Playground." Ideally it should be changed everywhere "MessageCard Playground" is mentioned.

Reply:
The url (https://messagecardplayground.azurewebsites.net) seems to be the same, correct?

Reply:
Correct

Reply:
One last thing -- in the playground, if you send the card via email, the subject line of the email reads "MessageCard Playground Test Card"   (I just tested it minutes ago.)

Reply:
Yes, known issue. Haven't had a chance to fix it yet.

Reply:
But in that case, are we supposed to update the module to reflect something that's not accurate in prod?

Reply:
Yes please. The documentation is already updated.